components/tools/ToolDashboard.js

```diff
@@ -109,6 +125,21 @@ export default function ToolDashboard() {
                         )}
                     </div>
                 </div>
+                <div className='flex w-1/3 lg:w-1/5 gap-5 h-auto'>
+                    <div className="relative w-full h-auto" ref={categoryRef}>
```

Comment:
I think the `ref` should be applied to the Dropdown menu only, not on the whole section of `Jump to Category`.